### PR TITLE
go/commands: add acme-lsp

### PR DESCRIPTION
**Pull request by rminnich**
Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

File: sys/src/cmd/go/commands [hunk 1]
```diff
@@ -1,3 +1,7 @@
 edwood git://github.com/Harvey-OS/edwood.git edwood
 tools/gopls git://github.com/golang/tools.git tools
 tools/cmd/goimports ... 
+acme-lsp/cmd/acme-lsp git://github.com/fhs/acme-lsp acme-lsp
+acme-lsp/cmd/acmefocused ...
```
Comment (fhs) [inline sys/src/cmd/go/commands:5]: I don't think `acmefocused` works on plan9. It uses unix domain socket.

**Review by fhs — APPROVED** [review]

**Comment by rminnich** [comment]
I won't commit this until someone else can verify that a usb stick still works. I'm having a problem.

**Comment by rminnich** [comment]
OK confirmed it works and that the programs are there. Now to learn how to use them on harvey.

**Review by gmacd — DISMISSED** [review]

**Comment by rminnich** [comment]
OK, back to 4G memory!

just built a usb with it.

**Review by floren — APPROVED** [review]

**Pull request merged**